### ISSUE TITLE
About qwen3.5 mtp sft, multi-modality data

### ISSUE THREAD

**Issue by guoyingying432** [modelscope/ms-swift#10057]
### Checklist / 检查清单

- [x] I have searched existing issues, and this is a new question or discussion topic. / 我已经搜索过现有的 issues，确认这是一个新的问题与讨论。

### Question Description / 问题描述

When I meagtron-sft the qwen3.5-2b model with the multi-modality data and turn on the mtp, the loss seems lower than only text data. 

{"loss": 0.04149855, "grad_norm": 1.89314604, "learning_rate": 6.2e-07, "mtp_1_loss": 0.06920646, "iteration": "1/3207", "elapsed_time": "2m 8s", "remaining_time": "4d 17h 46m 31s", "memory(GiB)": 62.88, "train_speed(s/it)": 127.757503}
{"loss": 0.06579629, "grad_norm": 2.2239275, "learning_rate": 3.12e-06, "mtp_1_loss": 0.1017431, "iteration": "5/3207", "elapsed_time": "3m 53s", "remaining_time": "1d 17h 22m 57s", "memory(GiB)": 62.89, "train_speed(s/it)": 46.526125}

**Comment by z0o0ey** [comment]
The MTP loss uses a simple average across micro-batches rather than a token-weighted average, and `loss_scale` is not applied.
